Marlin/chunk_support.cpp

```diff
@@ -0,0 +1,68 @@
+#include "Arduino.h"
+#include "Macros.h"
+#include "Configuration.h"
+#include "serial.h"
+#include "temperature.h"
+
+#define BLINK_LED LED_BUILTIN
+
+//wave tables, 4 bit move, +/- 7
+uint8_t block_moves[16][8] = {
```

Comment:
wave tables for each step frequency, done by hand. this is my third version for this encoding, it seems to be the least annoying noise wise.